### PR TITLE
[LOOP-5434] Solopump Demo Scenarios

### PR DESCRIPTION
**Pull request by Camji55**
https://tidepool.atlassian.net/browse/LOOP-4534

### CODE REVIEW

**Review by nhamming — APPROVED** [review]
LGTM

File: LoopTestingKit/TestingScenarioInstance.swift [hunk 1]
```diff
@@ -14,4 +14,12 @@ public struct TestingScenarioInstance {
     public var futureGlucoseSamples: [NewGlucoseSample]
     public var pumpEvents: [NewPumpEvent]
     public var carbEntries: [NewCarbEntry]
+    
+    public var hasCGMData: Bool {
+        !(pastGlucoseSamples + futureGlucoseSamples).isEmpty
+    }
+    
+    public var hasPumpData: Bool {
+        !pumpEvents.isEmpty || !carbEntries.isEmpty
```
Comment (ps2) [inline LoopTestingKit/TestingScenarioInstance.swift]: Are carbs considered pump data here for some reason?

Comment (Camji55) [reply LoopTestingKit/TestingScenarioInstance.swift]: I'll remove it :)

Comment (Camji55) [reply LoopTestingKit/TestingScenarioInstance.swift]: Removed

**Pull request merged**